Explorer/Assets/DCL/InWorldCamera/CameraReelGallery/CameraReelGalleryController.cs

```diff
@@ -81,16 +86,22 @@ public ReelToDeleteInfo(string id, string datetime)
         public CameraReelGalleryController(CameraReelGalleryView view,
             ICameraReelStorageService cameraReelStorageService,
             ICameraReelScreenshotsStorage cameraReelScreenshotsStorage,
-            IWebBrowser webBrowser,
-            IDecentralandUrlsSource decentralandUrlsSource,
-            IExplorePanelEscapeAction explorePanelEscapeAction,
-            ISystemClipboard systemClipboard,
-            string shareToXMessage,
-            string photoSuccessfullyDeletedMessage,
-            string photoSuccessfullyUpdatedMessage,
-            string linkCopiedMessage,
+            int gridLayoutFixedColumnCount,
+            int thumbnailHeight,
+            int thumbnailWidth,
+            bool gridShowMonth,
+            bool useSignedRequest,
```

Comment:
Maybe these could be in a struct called CameraReelGallerySettings or whatever? to make it a bit cleaner to read and setup? wdyt?